core/base/mergeTreeClustering/BranchMappingDistance.h

```diff
@@ -0,0 +1,522 @@
+/// \ingroup base
+/// \class BranchMappingDistance
+/// \author Florian Wetzels (wetzels@cs.uni-kl.de)
+/// \date 2022.
+///
+/// This module defines the %BranchMappingDistance class that computes distances
+/// between two merge trees.
+///
+/// \b Related \b publication \n
+/// "Branch Decomposition-Independent Edit Distances for Merge Trees." \n
+/// Florian Wetzels, Heike Leitte, and Christoph Garth.\n
+/// Computer Graphics Forum, 2022.
+
+#pragma once
+
+#include <set>
+#include <vector>
+
+#include <algorithm>
+#include <cfloat>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <set>
+#include <stack>
+#include <tuple>
+#include <vector>
+
+// ttk common includes
+#include <AssignmentAuction.h>
+#include <AssignmentExhaustive.h>
+#include <AssignmentMunkres.h>
+#include <Debug.h>
+
+namespace ttk {
+
+  class BranchMappingDistance : virtual public Debug {
+
+  private:
+    int baseMetric = 0;
```

Comment:
usually, in ttk, the member variables of a class, within the `core/base/` subdirectories, end with a final `_` (in order to distinguish them from local variables, later in the functions).
see `assignmentSolverID_` for instance.
that'd be great if you could follow that convention.

Reply:
Sure, I will have a look into that.